### DISCUSSION TITLE
forrtl: severe (59): list-directed I/O syntax error, unit 100

### DISCUSSION THREAD

**Discussion by AlvaroJPZ** [Anura3D/Anura3D_OpenSource#192] [Q&A]
Please, how can I solve this error:

forrtl: severe (59): list-directed I/O syntax error, unit 100, file E:\1.ALVARO\Archivos personales2\Terceros\Lavalin\Anura2\Sec3bs.A3D\Sec3bs.GOM
Image              PC                Routine            Line        Source
Anura3D_2023.exe   00007FF76A605965  Unknown               Unknown  Unknown
Anura3D_2023.exe   00007FF769DCE14F  Unknown               Unknown  Unknown
Anura3D_2023.exe   00007FF76A5D3D6A  Unknown               Unknown  Unknown
Anura3D_2023.exe   00007FF76A5D4C41  Unknown               Unknown  Unknown
Anura3D_2023.exe   00007FF76AABE3AB  Unknown               Unknown  Unknown
Anura3D_2023.exe   00007FF76EF5EB74  Unknown               Unknown  Unknown
KERNEL32.DLL       00007FFEDED07344  Unknown               Unknown  Unknown
ntdll.dll          00007FFEE07A26B1  Unknown               Unknown  Unknown

E:\1.ALVARO\Archivos personales2\Terceros\Lavalin\Anura2\Sec3bs.A3D>PAUSE

Thanks in advance.

**Comment by EmreDumlu-UM** [comment]
I believe this error caused by several reasons. 
First, I would say take a look at the input parameters  and re-consider that whether any of them creates a conflict. 
If you use double-point, it would be better to apply double point: define additional parameters. 
This error also comes if the just created Anura3D folders does not exist anymore in the  directory (such as .GOM file)

**Comment by AlvaroJPZ** [comment]
thank you very much. Some thing I realized, was that when I put "Generate Anura3D files", It was created two diferents folders: .A3D and .Gid. I dont know if that was correct.
At first there was an error because, for some reason, didnt copy A3DMohrCoulombStandard.dll, it was missed. So I copied manually. It could be that I did no compile right Anura3D? 
You are very kind for answer, I very new on this.

**Reply by EmreDumlu-UM** [reply]
If you once compile successfully Anura3D, you do not have to copy manually any .dll files from the main folder. If you watch the steps on the manual, I am sure you will able to calculate

**Reply by AlvaroJPZ** [reply]
I had a very dense mesh on some areas, that could be the reason for that error?

**Reply by EmreDumlu-UM** [reply]
You should be sure if you compile Anura3D succesfully based on the previous conversation. If you are sure, then you should look at the directory where all the generated Anura3D files via GiD exist.

**Reply by AlvaroJPZ** [reply]
I redid all, deleted some points in order to created not thath dense mesh. Again, the A3DMohrCoulombStandard.dll was not automatically copied when the files were created, but I copy manually and It works, the error didnt came.

I am running a runout  analysis, on dry conditions, may be you could help me defining all calculate data, I am not sure about the inicial conditions, my reference is the sliding blocks example from the manual.

**Reply by kobidpanthi** [reply]
> I redid all, deleted some points in order to created not thath dense mesh. Again, the A3DMohrCoulombStandard.dll was not automatically copied when the files were created, but I copy manually and It works, the error didnt came.
> 
> I am running a runout analysis, on dry conditions, may be you could help me defining all calculate data, I am not sure about the inicial conditions, my reference is the sliding blocks example from the manual.

Hello. Same thing happened to my model but i could not find any dll files as well. Where did you copy them from? Could you please help me out

**Reply by elbarrendero** [reply]
hello i'm using Anura3D_2025, and GiD 17, i'm getting the same errors...

**Comment by CH-max92** [comment]
Hi , AlvaroJPZ.
In this case, there is usually something wrong with the grid. It is also possible that there are some potential conflicts in the material parameters settings. As for the lost of the .dll file, I suppose that you didn't compile the code for anura3D properly.
